docs/perf_client.rst

```diff
@@ -235,20 +235,44 @@ type INT32: ::
     "data" :
      [
         {
-          "INPUT0" : [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
-          "INPUT1" : [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
+          "INPUT0" :
+                {
+                    "content": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
```

Comment:
This is not a backwards-compat change. Do you still handle the old format correctly?

Reply:
I removed the support for old format. Should I add the support for maintaining backwards compatibility?

Reply:
Yes, and keep 1 test that uses the old format.